source/renderer/app/components/wallet/receive/WalletReceiveRandom.js

```diff
@@ -217,6 +217,7 @@ export default class WalletReceiveRandom extends Component<Props, State> {
 
         <Button
           className={generateAddressButtonClasses}
+          disabled={isSubmitting || !passwordField.value}
```

Comment:
@DeeJayElly Please create variable for this check. Something like `canGenerate` or similar

Reply:
Done